codegens/curl/test/unit/convert.test.js

```diff
@@ -252,5 +252,72 @@ describe('curl convert function', function () {
         }
       });
     });
+
+    it('should not encode queryParam unresolved variables and ' +
+    'leave it inside double parenthesis {{xyz}}', function () {
+      request = new sdk.Request({
+        'method': 'POST',
+        'header': [],
+        'url': {
+          'raw': 'http://postman-echo.com/post?a={{xyz}}',
+          'protocol': 'http',
+          'host': [
+            'postman-echo',
+            'com'
+          ],
+          'path': [
+            'post'
+          ],
+          'query': [
+            {
+              'key': 'a',
+              'value': '{{xyz}}'
+            }
+          ]
+        }
+      });
+      options = {};
+      convert(request, options, function (error, snippet) {
+        if (error) {
+          expect.fail(null, null, error);
+        }
+        expect(snippet).to.be.a('string');
+        expect(snippet).to.include('http://postman-echo.com/post?a={{xyz}}');
```

Comment:
why can't we do an equality check here?

Reply:
equality for the whole generated snippet ?